test/passing/tests/break_separators-after.ml.ref

```diff
@@ -89,6 +89,14 @@ let _ =
       0
   | _ -> 1
 
+let _ =
+  match something with
+  | [: very_very_long_field_name_running_out_of_space;
```

Comment:
Elsewhere, the semicolons come at the beginning. Should I be concerned?

Reply:
What you should be reviewing is how the immutable arrays look relative to the other arrays _in this file_.  If we look at the name of this file, it's `break_separators-after.ml.ref`; this means it comes from `break_separators.ml`, *subject to the settings denoted by `after`*.  So in *this* file, we're testing that the separators come "after" the list elements!  You'll likely want to take another look at this and the similar files with that in mind.